Camera.MAUI/Platforms/Android/MauiCameraView.cs

```diff
@@ -255,6 +246,30 @@ internal async Task<CameraResult> StartRecordingAsync(string file, Microsoft.Mau
         return result;
     }
 
+    // 0=landscape left, 90=portrait(up), 180=landscape right, 270=upside down (rear cam)
+    static readonly int[] degrees = { 0, 90, 180, 270 };
+
+    /// <summary>
+    /// We want to map the raw device orientation to the expected window orientation in case app-orientation is locked.
+    /// Uses accelerometer to determine device orientation in degrees (0 = landscape with "top"/camera-end of phone to user-left)
+    /// </summary>
+    /// <returns></returns>
+    private int CheckOrientation()
+    {
+        float angle = _camService.DeviceOrientationAngle;
+        // Map angle to nearest 0, 90, 180, 270 degrees
+        uint index = (uint)Math.Round(angle / 90.0f) % 4u;
+        int deviceOrientation = degrees[index];
+        deviceOrientation = cameraView?.Camera?.Position == CameraPosition.Front
```

Comment:
Hah, actually,  `GetJpegOrientation()` does this already.

```csharp
        var cameraPosition = cameraView.Camera.Position == CameraPosition.Front ? -1 : 1;
        return (sensorOrientation - deviceOrientation * cameraPosition + 360) % 360
```

Reply:
Just for grins, I tried this out for correcting video orientation instead of my new impl.  There's something not quite right with this code, so probably the same issue exists for still-pics as well, but we can hold that for another issue/PR.  Possibly we could take whatever we settle on for video as the fix for this?

Reply:
This issue in `GetJpegOrientation()` is that it is still tied to `windowManager.DefaultDisplay.Rotation` which only tracks with the device if the app config is set to re-orient with rotation.